content/rs/references/cli-utilities/crdb-cli/crdb/flush.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: crdb-cli crdb flush
+linkTitle: flush
+description: Clears all keys from an Active-Active database.
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+aliases:
+---
+
+`crdb-cli crdb flush` clears all keys from an Active-Active database.
+
+```sh
+crdb-cli crdb flush --crdb_guid <guid>
+                    [ --no-wait ]
+```
+
+### Parameters
+
+| Parameter           | Value  | Description                         |
+|---------------------|--------|-------------------------------------|
+| crdb_guid \<guid\>  | string | The GUID of the database (required) |
+| no-wait             |        | Does not wait for the task to complete |
+
+### Returns
+
+Returns the task ID of the task that is clearing the database. If --no-wait is specified, the command exits. Otherwise, it will wait for the database to be cleared and return `finished`.
```

Comment:
```suggestion
Returns the task ID of the task clearing the database. If `--no-wait` is specified, the command exits. Otherwise, it will wait for the database to be cleared and return `finished`.
```